config/migrations/Version20190425071843.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Platforms\MariaDb1027Platform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+final class Version20190425071843 extends AbstractMigration implements ContainerAwareInterface
```

Comment:
I think better rename to something like "SetupEventStoreTables"

Reply:
[Doctrine documentation recommends](https://www.doctrine-project.org/projects/doctrine-migrations/en/2.0/reference/version-numbers.html) keeping migrations classes naming in that format, to maintain versioning of migrations more easily.

Reply:
I'm not sure this is applies here. As this is an event sourced application, there are no migrations at all. You would simply destroy your read model and create a new one.
Migrations here makes only sense as a way to setup the event store. Not to add 'future' migrations.

Reply:
From that point of view, I agree with you, renamed